src/components/employee/employee-history/employee-history-component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from 'angular2/core';
+import {AppliedLeave} from '../../../domains/applied-leave';
+import {LeaveService} from '../../../services/leave-service';
+
+@Component({
+  selector: 'employee-history',
+  templateUrl : `employee-history.tpl.html`,
+  providers: [LeaveService]
```

Comment:
We are adding all service providers in the main MisFrontend component. So please remove this.